test/unit/spec/v2/ion/i18nTransformer_spec.js

```diff
@@ -28,6 +28,7 @@ describe('v2/ion/i18nTransformer', function() {
       'enroll.onprem.username.placeholder': 'enter {0} username',
       'enroll.onprem.passcode.placeholder': 'enter {0} passcode',
       'errors.E0000106': 'token change mode',
```

Comment:
remove `errors.E0000106` ?

Reply:
106 is still used for enrollment case.. so i left it there